core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -295,9 +329,9 @@ export class OptimizedSsrEngine {
     });
   }
 
-  protected log(message: string, debug = true): void {
-    if (!debug || this.ssrOptions?.debug) {
-      console.log(message);
+  protected log(message: string, logMetadata?: LogContext, debug = true): void {
```

Comment:
[MINOR]

Formally, we made a breaking change here - changed type of the 2nd argument. 
To fix it, we would need to add new param as the 3rd one.

Side note: I'm not fond of the current signature of this method, (especially with debug: boolean), so in the future we might refactor it anyway.

Reply:
Do we consider this a breaking change? correct me if I'm wrong, but AFAIK `OptimizedSsrEngine` is not a part of the public API, so I thought it doesn't change that much.